src/SecurityDetails/SecurityDetailsOptions.ts

```diff
@@ -59,8 +68,17 @@ export interface SecurityDetailsMetadata extends MorningstarMetadata {
 export interface SecurityDetailsOptions extends MorningstarOptions {
     security?: MorningstarSecurityOptions,
     viewId?: string,
+    converter?: SecurityDetailsConverterOptions
```

Comment:
This is just an idea: You might consider (in the future) to also support multiple converters simultaneously and move the `table.deleteColumns` to the connector. The abstract `MorningstarConverter` then will need a second constructor argument for a shared connector-table to start with.